docker-compose-fullstack.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
```

Comment:
I find external ports allow devs to play around more easily with services, like being able to drop into SQL shells or test locally with parts of a stack.

Celery - is this currently being used in prod? Might be good to add it in another issue.